app/controllers/followers_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# followers_controller.rb
+class FollowersController < ApplicationController
+  before_action :authorized
+
+  def create
+    follow = Follower.create(follow_params)
+    follow.save
```

Comment:
You are saving the object twice.

Reply:
```
follow = Follower.new(follow_params)
follow.save if follow.valid?
redirect_article
```
Is true, now I am creating the object and saving